src/metabase/server/handler.clj

```diff
@@ -1,26 +1,46 @@
 (ns metabase.server.handler
   "Top-level Metabase Ring handler."
-  (:require [metabase.config :as config]
-            [metabase.plugins.classloader :as classloader]
-            [metabase.server.middleware.auth :as mw.auth]
-            [metabase.server.middleware.browser-cookie :as mw.browser-cookie]
-            [metabase.server.middleware.exceptions :as mw.exceptions]
-            [metabase.server.middleware.json :as mw.json]
-            [metabase.server.middleware.log :as mw.log]
-            [metabase.server.middleware.misc :as mw.misc]
-            [metabase.server.middleware.offset-paging :as mw.offset-paging]
-            [metabase.server.middleware.security :as mw.security]
-            [metabase.server.middleware.session :as mw.session]
-            [metabase.server.middleware.ssl :as mw.ssl]
-            [metabase.server.routes :as routes]
-            [ring.middleware.cookies :refer [wrap-cookies]]
-            [ring.middleware.gzip :refer [wrap-gzip]]
-            [ring.middleware.keyword-params :refer [wrap-keyword-params]]
-            [ring.middleware.params :refer [wrap-params]]))
+  (:require
+   [metabase.config :as config]
+   [metabase.server.middleware.auth :as mw.auth]
+   [metabase.server.middleware.browser-cookie :as mw.browser-cookie]
+   [metabase.server.middleware.exceptions :as mw.exceptions]
+   [metabase.server.middleware.json :as mw.json]
+   [metabase.server.middleware.log :as mw.log]
+   [metabase.server.middleware.misc :as mw.misc]
+   [metabase.server.middleware.offset-paging :as mw.offset-paging]
+   [metabase.server.middleware.security :as mw.security]
+   [metabase.server.middleware.session :as mw.session]
+   [metabase.server.middleware.ssl :as mw.ssl]
+   [metabase.server.routes :as routes]
+   [ring.core.protocols :as ring.protocols]
+   [ring.middleware.cookies :refer [wrap-cookies]]
+   [ring.middleware.gzip :refer [wrap-gzip]]
+   [ring.middleware.keyword-params :refer [wrap-keyword-params]]
+   [ring.middleware.params :refer [wrap-params]]))
```

Comment:
This is the new way to indent `:require`s. New rule is to reindent stuff this way as you make changes to stuff